.github/workflows/nodejs-perf.yml

```diff
@@ -0,0 +1,104 @@
+name: Node.js Performance
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+    paths:
+      - 'binding/nodejs/perf.test.js'
+      - 'lib/common/**'
+      - 'lib/node/**'
+      - '.github/workflows/nodejs-perf.yml'
+
+  pull_request:
+    branches: [ main, 'v[0-9]+.[0-9]+' ]
+    paths:
+      - 'binding/nodejs/perf.test.js'
+      - 'lib/common/**'
+      - 'lib/node/**'
+      - '.github/workflows/nodejs-perf.yml'
+
+defaults:
+  run:
+    working-directory: binding/nodejs
+
+jobs:
+  perf-github-hosted:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        include:
+          - os: ubuntu-latest
+            num_test_iterations: 50
+            init_performance_threshold_sec: 0.001
```

Comment:
I'm not sure we should even keep the init test in - I don't think it adds any value. If we decide to add it, we can add it to all device based animals